pkg/stagemanager/stage_manager.go

```diff
@@ -567,3 +568,36 @@ func (stm *StageManager) RunAll() {
 	// stop working
 	stm.Stop()
 }
+
+// StopMosnProcess stops Mosn process via command line,
+// and it not support stop on windows.
+func (stm *StageManager) StopMosnProcess() (err error) {
+	// init
+	stm.runParamsParsedStage()
+	stm.runInitStage()
+
+	mosnConfig := stm.data.config
+
+	// reads mosn process pid from `mosn.pid` file.
+	var pid int
+	if pid, err = pf.GetPidFrom(mosnConfig.Pid); err != nil {
+		log.StartLogger.Errorf("[mosn stop] fail to stop MOSN, get pid: %v \n", err)
+		time.Sleep(100 * time.Millisecond) // waiting logs output
+		return
+	}
+
+	// sends SIGTERM to mosn process, makes it graceful exit.
+	proc, err := os.FindProcess(pid)
+	// check if process is existing.
+	err = proc.Signal(syscall.Signal(0))
+	if err != nil {
+		log.StartLogger.Errorf("[mosn stop] fail to stop MOSN, process is not existing, err [%v] \n", err)
+		time.Sleep(100 * time.Millisecond) // waiting logs output
+		return
+	}
+
+	proc.Signal(syscall.SIGINT)
+	log.StartLogger.Infof("[mosn stop] has sent an INT signal to %v pid. \n", pid)
```

Comment:
```suggestion
	log.StartLogger.Infof("[mosn stop] sending INT signal to process(%v)", pid)
```
also, move this line before sending signal.

Reply:
okay